server/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -119,6 +124,7 @@ private static void handleException(HttpServletResponse response, ObjectMapper o
   private final RequestLogger requestLogger;
   private final GenericQueryMetricsFactory queryMetricsFactory;
   private final AuthenticatorMapper authenticatorMapper;
+  private final transient ProtobufTranslation protobufTranslation;
```

Comment:
The servlet is serializable and the ProtobufTranslation isn't, so this is required

Reply:
Hmm, I didn't know that the servlet is serializable. However, [it seems that Jetty doesn't support serializing servlets anyway](https://github.com/eclipse/jetty.project/issues/5169). I would suggest to not add it since it can be misleading as other variables are not transient.

Reply:
After removing `transient` spotbugs is failing with 
```
org.apache.calcite.avatica.remote.ProtobufTranslationImpl stored into non-transient field
```

Reply:
You can add an exclusion rule for it [here](https://github.com/apache/druid/blob/master/codestyle/spotbugs-exclude.xml#L40).